spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.feature 'Post Index', type: :feature do
+  let(:user) { User.create(name: 'Tom', photo: 'https://www.kasandbox.org/programming-images/avatars/leaf-blue.png', bio: 'He is a good programmar') }
+  let!(:post) { Post.create(author: user, title: "first post's title", text: 'first text') }
+  let!(:comment1) { Comment.create(author: user, post:, text: 'first comment') }
+  let!(:comment2) { Comment.create(author: user, post:, text: 'second comment') }
+  let!(:comment3) { Comment.create(author: user, post:, text: 'third comment') }
+  let!(:like1) { Like.create(author: user, post:) }
+
+  scenario "see user's profile picture, username, number of posts and interactions" do
+    visit user_posts_path(user)
+    expect(page).to have_content('Tom')
+    expect(page).to have_content(' posts')
+    expect(page).to have_content('Comments: 3')
+    expect(page).to have_content('Likes: 1')
+  end
+
+  scenario "see some of the post's title, body and first comments" do
+    visit user_posts_path(user)
+    expect(page).to have_content('first text')
+    expect(page).to have_content('first comment')
+    expect(page).to have_content("first post's title")
+  end
+
+  scenario 'see a section for pagination if there are more posts than fit on the view' do
+    visit user_posts_path(user)
+    expect(page).to have_selector('.pagination')
+  end
+
```

Comment:
- [ ] Dear Team, here also we are missing the test for `tests that assert you can see the user's profile picture.`,  and `Tests that assert you can see the number of posts the user has written.`